src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,11 @@
 // your components will all go in this `component` directory.
 // feel free to change this component.js into TodoList.js
+import React from 'react';
+
+const TodoList = props => {
```

Comment:
An option is to iterate the array of tasks here using the information passed down as props from the App component. You may also import *Todo*  here and use it for each iteration.